src/bt-core-io.c

```diff
@@ -523,9 +523,17 @@ discovery_state_changed_cb(bt_discovery_state_t state)
   cleanup_pdu_wbuf(wbuf);
 }
 
+/* Bug-1236841: CAF headers have different signature for this function,
+ * so if Q_BLUETOOTH is defined, we use the CAF one */
```

Comment:
I think this comment rather belongs into the Android.mk file. Q_BLUETOOTH might have an effect in other places as well. Nits: no '-' after 'Bug' and period at the end of the sentence